rustls-post-quantum/src/lib.rs

```diff
@@ -127,6 +127,10 @@ impl SupportedKxGroup for X25519Kyber768Draft00 {
     fn name(&self) -> NamedGroup {
         NAMED_GROUP
     }
+
+    fn usable_for_version(&self, version: ProtocolVersion) -> bool {
+        u16::from(version) >= u16::from(ProtocolVersion::TLSv1_3)
```

Comment:
Is this an instance where using `==` instead of `>=` would be akin to "failing closed" and might be better?

Reply:
I guess maybe writing code now that works properly for TLS1.4 is probably premature by a decade or so. Will replace with a straight equality.